build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\installers\create-installers.targets" />
+  <PropertyGroup>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PkgInstallDir Condition="$([MSBuild]::IsOsPlatform(Linux))">/usr</PkgInstallDir>
+    <PayloadDir>$(OutputPath)/pkgarchive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)/packages</PkgOutputPath>
+  </PropertyGroup>
+  <Target Name="_CopyFilesToPayloadDir"
+      DependsOnTargets="GetXAVersion" >
+    <PropertyGroup>
+      <XAFrameworkDir>$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions/$(XAVersion)</XAFrameworkDir>
+      <MonoFrameworkExternalDir>$(PayloadDir)/Library/Frameworks/Mono.framework/External</MonoFrameworkExternalDir>
+      <XBuildDir>$(XAFrameworkDir)/lib/xamarin.android/xbuild/Xamarin/Android</XBuildDir>
```

Comment:
We should probably start expunging "xbuild" from anything meaningful, as we haven't supposed `xbuild` in ages, and the directory name is just...what we've done.

We should call this `$(MSBuildTargetsDir)`.